app/services/power_manager/base.rb

```diff
@@ -17,6 +32,24 @@ def logger
       Rails.logger
     end
 
+    protected
+
+    def default_action(action)
+      raise NotImplementedError.new
+    end
+
+    def action_map
+      {
+        :status => { :output => :translate_status }
+      }
+    end
+
+    def translate_status(result)
+      return 'on' if result.to_lower.match(/on/)
```

Comment:
Nitpick, `result.match(/off/i)` will match without having to transform the string